docker-plain/files/chart/templates/_affinity.tpl

```diff
@@ -0,0 +1,51 @@
+{{/*
+Part of the ODS helm tpl library
+
+Version: 1.0
+*/}}
+
+
+{{/*
+Pod affinity/anti-affinity (soft)
+
+Usage: Include where needed, e.g.
+````
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  template:
+    spec:
+      affinity:
+        podAntiAffinity: {{- include "common.affinities.pods.soft" . | nindent 10}}
```

Comment:
Is there a specific reason for having _affinity.tpl and _labels.tpl as separate files even though there definition both start with 'common'? Would it make sense to put them together into a _common.tpl?

Reply:
IMHO I would keep it as proposed, as this is based on good practise templating in helm (based on the concept in coding in general, to split code/project source by context), in one template we handle affinity in another labels and so on. This helps keep complexity low.

Reply:
Hehe that was actually my point as well when I proposed the change. IMO, currently, there is a _helpers.tpl (not good splitted/named by context btw. 😆) in which the definitions all start with 'chart'. So I see 'chart' as the context here. For the definitions inside _affinity.tpl as well as _labels.tpl they all start with 'common', so I guess the context here is common?